### PR TITLE
Bedrock Components

### PR DESCRIPTION
**Pull request by 00salmon**
## Description of the change

> Low level widget/portal components, usually containers that provide additional functionality to components rather than any style/display effects.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation or Development tools (readme, specs, tests, code formatting)

## Links

 - Jira issue number: https://saasquatch.atlassian.net/browse/DEV-2996
 - Process.st launch checklist: https://app.process.st/checklists/Bedrock-Components-isnRzLzleI2CFx1B1KVP_A

### Development

- [ ] [Prettier](https://www.npmjs.com/package/prettier) was run
- [ ] The behaviour changes in the pull request are covered by specs
- [ ] All tests related to the changed code pass in development

### Paperwork

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has a Jira number
- [ ] This pull request has a Process.st launch checklist
- [ ] "Ready for review" label attached to the PR and reviewers pinged on Slack

### Code review 

- [ ] Changes have been reviewed by at least one other engineer
- [ ] Security impacts of this change have been considered

### CODE REVIEW

File: packages/bedrock-components/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@saasquatch/bedrock-components",
+  "version": "0.0.2",
```
Comment (00salmon) [inline packages/bedrock-components/package.json:3]:
```suggestion
  "version": "1.0.0",
```

File: packages/bedrock-components/package.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  "collection:main": "dist/collection/index.js",
+  "unpkg": "dist/bedrock-components/bedrock-components.esm.js",
+  "files": [
+    "dist/",
+    "loader/",
+    "docs/"
+  ],
+  "license": "MIT",
+  "author": "SaaSquatch Team",
+  "repository": "https://github.com/saasquatch/program-tools/tree/master/packages/bedrock-components",
```
Comment (loganvolkers) [inline packages/bedrock-components/package.json:20]: @00salmon Repository field isn't quite right

File: packages/bedrock-components/package.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  "scripts": {
+    "build": "stencil build --docs",
+    "start": "stencil build --dev --watch --serve",
+    "test": "stencil test --spec --e2e",
+    "test.watch": "stencil test --spec --e2e --watchAll",
+    "generate": "stencil generate"
+  },
+  "dependencies": {
+    "@saasquatch/component-boilerplate": "^1.0.0-13",
+    "@saasquatch/stencil-grapes-plugin": "^0.1.2",
```
Comment (loganvolkers) [inline packages/bedrock-components/package.json:30]: @00salmon Should use the 1.0 versions

File: packages/bedrock-components/src/components/sqb-conditional-section/sqb-conditional-section.feature [hunk 4]
```diff
@@ -0,0 +1,48 @@
+@owner:sam
+@author:sam
+
+Feature: Conditional Section
+
+  Conditional Section adds JSONata conditional logic that can display or hide children components based on user segments or custom fields
+
+  @review
```
Comment (loganvolkers) [inline packages/bedrock-components/src/components/sqb-conditional-section/sqb-conditional-section.feature:8]: Remove all the `@review` tags.

```suggestion
```

Comment (loganvolkers) [inline packages/bedrock-components/src/components/sqb-conditional-section/sqb-conditional-section.feature:8]: Remove all the `@review` tags.

```suggestion
```

**Review by loganvolkers — CHANGES_REQUESTED** [review]
Version number update and package.json cleanliness

File: packages/bedrock-components/package.json [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  "scripts": {
+    "build": "stencil build --docs",
+    "start": "stencil build --dev --watch --serve",
+    "test": "stencil test --spec --e2e",
+    "test.watch": "stencil test --spec --e2e --watchAll",
+    "generate": "stencil generate"
+  },
+  "dependencies": {
+    "@saasquatch/component-boilerplate": "^1.0.0",
+    "@saasquatch/stencil-grapes-plugin": "^1.0.0",
```
Comment (loganvolkers) [inline packages/bedrock-components/package.json:34]: This should be a dev dependency

```suggestion
    "@saasquatch/stencil-grapes-plugin": "^1.0.0",
```

**Review by loganvolkers — APPROVED** [review]
Looks good to me!

**Pull request merged**